ROADMAP.md

```diff
@@ -7,24 +7,38 @@ quarterly.  Podman maintainers then prioritize these features.   Top features ar
 one or more engineers.
 
 
-## Future feature focus
+## Future feature considerations
+
+The following features are of general important to Podman.  While these features have no timeline
+associated with them yet, they will likely be on future quarterly milestones.
 
 * Further improvements to `podman machine` to better support Podman Desktop and other developer usecases.
   - Smoother upgrade process for Podman machine operating system (OS) images
-  - Release and pipeline automation for Podman machine OS images
-  - Performance improvements
   - Convergence of WSL technologies with other providers including its OS
-* OCI artifact support
+* Remote client remote of OCI and its RESTFUL API
 * Integration of composefs
 * Partial pull support (zstd:chunked)
 * Improved support for the BuildKit API.
 * Performance and stability improvements.
 * Reductions to the size of the Podman binary.
 
-## Previous quarterly feature prioritizations
+## Milestones and commitments by quarter
 
 This section is a historical account of what features were prioritized by quarter.  Results of the prioritization will be added before at the beginning of each quarter (Jan, Apr, July, Oct).
 
-**2025 Q1**
+### 2025 Q1 ####
+
+#### Releases ####
+- [x] Release Podman 5.4
+- [x] Podman release automation
+
+#### Features ####
+- [x] Artifact add --append
```

Comment:
Is there any value in providing links to items when they are completed?

Reply:
hi @jakecorrenti for historical purposes, someone might find a use for it.  And I contemplated just doing it.  But it is a duplication (if not double duplication) and we always have git log and git for this sort of thing ... which in fact is the source of truth.  So I was kind of thinking of a wait and see on this one ... also, when we implement features, we dont always do so in a single PR and now i'm not sure what to do with it.  We could also link JIRA cards, but that can change in in a heartbeat between deletion, duplication, or archival.

Reply:
All good points. I didn't have a preference either way, I was just curious.